packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.html

```diff
@@ -0,0 +1,36 @@
+<div data-ng-controller="IpSliceByoipCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="$ctrl.cancelSlice.bind($ctrl)"
+        data-wizard-on-finish="$ctrl.slice.bind($ctrl)"
+        data-wizard-title="('ip_byoip_slice_block_title' | translate) + ' ' + $ctrl.ip.ipBlock"
+    >
+        <div data-wizard-step data-wizard-step-valid="$ctrl.isLoaded">
+            <div data-ng-show="$ctrl.isLoaded">
+                <oui-message data-type="error" data-ng-show="$ctrl.errorMessage" >
+                    <span data-ng-bind="$ctrl.errorMessage"></span>
+                </oui-message>
+                <p data-translate="ip_byoip_slice_set_children_size"></p>
+                <label class="oui-select">
+                    <select data-ng-model="$ctrl.selectedSize" class="oui-select__input" >
+                        <option data-ng-repeat="size in $ctrl.ipSizes track by $index" data-ng-value="size">
+                            <span>{{'/' + size.slicingSize}}</span>
```

Comment:
```suggestion
                            <span data-ng-bind="'/' + size.slicingSize"></span>
```

Reply:
I didn't see this was inside an option tag. the data-ng-bind property should be on the option tag itself and you can remove the span inside

Reply:
done